src/main/resources/application.yaml

```
@@ -11,4 +11,15 @@ spring:
     password: ${DB_PASSWORD:password}
     driver-class-name: com.mysql.cj.jdbc.Driver
   profiles:
-    active: ${SPRING_PROFILE:development}
\ No newline at end of file
+    active: ${SPRING_PROFILE:development}
+  mail:
+    host: ${MAIL_HOST:smtp.gmail.com}
+    port: ${MAIL_PORT:587}
+    username: ${MAIL_USERNAME:7up.megastore@gmail.com}
+    password: ${MAIL_PASSWORD:kemb pmbj helh ikvn}
```

Ojo con estos valores por defectos que estás seteando ya que solo serviría en un ambiente de desarrollo, si el día de mañana esto se hosteara en un ambiente productivo y por alguna razon no se setean las variables, Spring va a tomar los valores que vos definiste por defecto acá lo cual estaría mal. Para evitar esto podes usar el application-development.yaml definiendo las mismas propiedades con los valores que estás definiendo por defecto acá, este archivo entonces te quedaría como

```suggestion
  mail:
    host: ${MAIL_HOST}
    port: ${MAIL_PORT}
    username: ${MAIL_USERNAME}
    password: ${MAIL_PASSWORD}
```

Pero a su vez tendrias el application-development.yaml así 

```
  mail:
    host: smtp.gmail.com
    port: 587}
    username: 7up.megastore@gmail.com
    password: kemb pmbj helh ikvn
```

De esta forma cuando levantas el proyecto en modo development, va a usar esos valores que definiste en el perfil development